src/pages/workspaces/workspace/ShareWorkspaceModal.js

```diff
@@ -87,7 +88,7 @@ export default ajaxCaller(class ShareWorkspaceModal extends Component {
   }
 
   render() {
-    const { onDismiss } = this.props
+    const { workspace, onDismiss } = this.props
```

Comment:
Why not destructure `accessLevel` here?

Reply:
It would need to be renamed to avoid conflicting with the state value. This seemed a tiny bit more readable.

Reply:
Fair enough. I say merge it.